### ISSUE TITLE
Unknown items in  tab list

### ISSUE THREAD

**Issue by Yeldece** [UnigramDev/Unigram#2404]
**Describe the bug**
When you delete a chat, Two items appear in the tab list. They are right before the button labelled as open navigation menu. The first item has text "chat deleted" on it which I read it via special screen reader cursor, item is literaly empty and unknown while navigating with tab.
Second is a button no label on it, clicking on it also doesn't do anything.

**To Reproduce**
Delete a chat and navigate with tab.

**Version Info**
Windows 10 20H2 (64-bit) build 19042.985
Figured out in latest unigram beta, which is  Unigram 7.8 (6506) X64 Debug
but quite sure it has been like this in older versions too.

**Comment by MarcAnt01** [comment]
@Yeldece thanks for reporting, but you need to fill version info properly

**Comment by Yeldece** [comment]
done
